src/workerd/api/pyodide/pyodide.c++

```diff
@@ -67,9 +67,15 @@ int ReadOnlyBuffer::read(jsg::Lock& js, int offset, kj::Array<kj::byte> buf) {
   return readToTarget(source, offset, buf);
 }
 
-kj::Array<jsg::JsRef<jsg::JsString>> PyodideMetadataReader::getNames(jsg::Lock& js) {
+kj::Array<jsg::JsRef<jsg::JsString>> PyodideMetadataReader::getNames(
+    jsg::Lock& js, kj::Maybe<kj::String> maybeExtFilter) {
```

Comment:
Why don't we go with jsg::Optional and use `maybeExtFilter?: string` as the type? 

```suggestion
    jsg::Lock& js, jsg::Optional<kj::String> maybeExtFilter) {
```

Reply:
I'd rather have the caller be explicit when calling this method.